src/plugins/intel_cpu/src/nodes/kernels/jit_kernel_base.hpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include <math.h>
+#include <dnnl_extension_utils.h>
+#include <cpu/x64/jit_generator.hpp>
+#include <cpu/x64/cpu_isa_traits.hpp>
+#include "jit_base.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace dnnl::impl;
```

Comment:
Please don't use *'using namespace'* in header files.

Reply:
done